core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/codeGen/ReplCodeGeneratorImpl.kt

```diff
@@ -68,7 +68,7 @@ internal class ReplCodeGeneratorImpl : ReplCodeGenerator {
                         // property scheme is valid for current data frame, but we should also check that all compatible open markers are implemented by it
                         val requiredBaseMarkers =
                             getRequiredMarkers(columnSchema, registeredMarkers.values)
-                        if (requiredBaseMarkers.all { currentMarker.implements(it) }) {
+                        if (requiredBaseMarkers.any() && requiredBaseMarkers.all { currentMarker.implements(it) }) {
```

Comment:
is this a fix for some bug? would be nice to have a test to see why this code was changed (or maybe there is? i didn't find it)

Reply:
yup. Accessors wouldn't be generated for my interface-nested DataSchemas without this fix. I think it was just a misinterpretation of `.all {}` again, that's why I didn't write an explicit test here, but the OpenApi Jupyter test breaks if this `.any()` check isn't there.